src/test/java/org/scijava/script/ScriptInfoTest.java

```diff
@@ -285,6 +287,11 @@ public ScriptEngine getScriptEngine() {
 		public List<String> getExtensions() {
 			return Arrays.asList("bsizes");
 		}
+
+        @Override
+        public AutoCompleter getAutoCompleter(ScriptEngine scriptEngine) {
+            return new DefaultAutoCompleter(this, scriptEngine);
```

Comment:
Weird. What is the warning? The whole point of having methods implemented in the abstract base class is so that you _don't_ have to implement them that way same in the subclass. NetBeans warning about that does not make sense...